projects/m2/008-word-by-word-palindromes/solutions/enricozoboli/solution_word_by_word_palindromes.py

```diff
@@ -0,0 +1,33 @@
+def remove_punctuation(s):
```

Comment:
As soon as I merge this, you should be able to import this function from the other exercise